src/org/pentaho/di/trans/steps/hadoopfileoutput/HadoopFileOutputMeta.java

```diff
@@ -26,11 +26,13 @@
 import org.pentaho.di.core.exception.KettleException;
 import org.pentaho.di.core.hadoop.HadoopSpoonPlugin;
 import org.pentaho.di.core.namedcluster.NamedClusterManager;
+import org.pentaho.di.core.variables.Variables;
 import org.pentaho.di.core.xml.XMLHandler;
 import org.pentaho.di.i18n.BaseMessages;
 import org.pentaho.di.repository.ObjectId;
 import org.pentaho.di.repository.Repository;
 import org.pentaho.di.trans.steps.textfileoutput.TextFileOutputMeta;
+import org.pentaho.di.ui.spoon.Spoon;
```

Comment:
@e-cuellar this is not going to work in many environments, Spoon / UI will not be allowed at runtime